lib/chef/provider/windows_task.rb

```diff
@@ -20,26 +20,153 @@
 require "rexml/document"
 require "iso8601"
 require "chef/mixin/powershell_out"
+require "chef/provider"
+require "win32/taskscheduler"
+require 'pry'
```

Comment:
Probably want to remove this one

Reply:
yes the coiding is in progress so I will be removing those while cleaning up the unnecessary code